src/components/Item.js

```diff
@@ -0,0 +1,24 @@
+import "./Item.css";
+import favoriteIcon from "../assets/icon/ic_favorite.svg";
+
+function Item({ item }) {
+  const { name, images, price, favoriteCount } = item;
+  const imgSrc = images[0];
```

Comment:
변수로 받지 않고 바로 images[0]을 넣어주셔도 좋습니다.

Reply:
가독성이 더 좋을 것 같아서 할당했는데, 큰 차이 없을 듯 해서 바로 넣었습니다.
이렇게 수정하는 것 말씀하신 게 맞나요?? 아니면 images를 받는 대신 바로 images[0]을 받을 수 있다는 뜻일까요?

[refactore(Item-imgSrc): Use images[0] directly instead of assigning it to variable imgSrc](https://github.com/un0211/6-Sprint-Mission/commit/368f087a2fd239beb804a04a070f3f8d8ccc7ffe)

Reply:
지금처럼 작성해주신것이 맞습니다!